### PR TITLE
docs: update ios sdk references

### PR DESCRIPTION
**Pull request by cre8ivejp**

### CODE REVIEW

File: docs/sdk/client-side/ios/index.md [hunk 1]
```diff
@@ -237,13 +238,28 @@ You don't need to call this method manually in regular use because the SDK is po
...
 <TabItem value="swift" label="Swift">
 
 ```swift showLineNumbers
-// TODO
+// Receiving notification in background
+func application(
+  _ application: UIApplication,
+  didReceiveRemoteNotification userInfo: [AnyHashable: Any]
+) async -> UIBackgroundFetchResult {
+  if userInfo["bucketeer_feature_flag_updated"] == "true" {
```
Comment (s2mr) [inline docs/sdk/client-side/ios/index.md:252]:
```suggestion
  let flag = userInfo["bucketeer_feature_flag_updated"] as? String
  if flag == "true" {
```

File: docs/sdk/client-side/ios/index.md [hunk 2]
```diff
@@ -263,6 +279,8 @@ This method lets you save user actions in your application as events. You can co
 
 In addition, you can pass a double value to the goal event. These values will sum and show as <br />`Value total` on the experiments console UI. This is useful if you have a goal event for tracking how much a user spent on your application buying items, etc.
 
+The default track value is 0.0.
+
 <Tabs>
 <TabItem value="swift" label="Swift">
```
Comment (s2mr) [inline docs/sdk/client-side/ios/index.md:286]: below code occur compile error.
use this.

```
client.track(goalId: "YOUR_GOAL_ID", value: 10.50)
```

Comment (s2mr) [reply docs/sdk/client-side/ios/index.md:286]:
```diff
- client.flush()
+ client.flush(completion: nil)
```

File: docs/sdk/client-side/ios/index.md [hunk 3]
```diff
@@ -304,14 +322,14 @@ This feature will give you robust and granular control over what users can see o
...
   "os_version": "11.0.0",
-  "device_model": "pixel-5"
+  "device_model": "iphone-12",
   "language": "english",
-  "genre": "female",
+  "genre": "female"
 ]
-
-let user = BKTUser(id: "USER_ID", attributes)
+let user = try! BKTUser(id: "USER_ID", attributes: attributes)
```
Comment (s2mr) [inline docs/sdk/client-side/ios/index.md:329]:
```suggestion
let user = try! BKTUser.create(id: "USER_ID", attributes: [:])
```

File: docs/sdk/client-side/ios/index.md [hunk 4]
```diff
@@ -362,20 +380,54 @@ let user = client.currentUser()
...
+    let client = BKTClient.shared
+    let showNewFeature = client.boolVariation(featureId: "YOUR_FEATURE_FLAG_ID", defaultValue: false)
+    if (showNewFeature) {
+      // The Application code to show the new feature
+    } else {
+      // The code to run when the feature is off
+    }
+  }
+}
+// Returned value is used when you want to remove listener
```
Comment (s2mr) [inline docs/sdk/client-side/ios/index.md:422]: Please add this line

```
let listener = EvaluationUpdateListenerImpl()
```

File: docs/sdk/client-side/ios/index.md [hunk 5]
```diff
@@ -362,20 +380,54 @@ let user = client.currentUser()
...
+let key = client.addEvaluationUpdateListener(listener: listener)
+
+// Remove a listener associated with the key
+client.removeEvaluationUpdateListener(key)
```
Comment (s2mr) [inline docs/sdk/client-side/ios/index.md:426]: missing argument label.

```suggestion
client.removeEvaluationUpdateListener(key: key)
```

File: docs/sdk/client-side/ios/index.md (hunk 1 above)
Comment (s2mr) [reply docs/sdk/client-side/ios/index.md:252]: It fixed compile error

File: docs/sdk/client-side/ios/index.md [hunk 6]
```diff
@@ -58,13 +58,13 @@ Configure the SDK config and user configuration.
...
   apiKey: "YOUR_API_KEY",
-  apiURL: "YOUR_API_URL",
-  featureTag: "YOUR_FEATURE_TAG"
+  apiEndpoint: "YOUR_API_ENDPOINT",
+  featureTag: "ios",
+  appVersion: bundle.infoDictionary?["CFBundleShortVersionString"] as! String
 )
-
-let user = BKTUser(id: "USER_ID")
+let user = try !BKTUser(id: "USER_ID")
```
Comment (s2mr) [inline docs/sdk/client-side/ios/index.md:67]: Error occurred.
fix keyword `!` position.

```suggestion
let user = try! BKTUser(id: "USER_ID")
```

File: docs/sdk/client-side/ios/index.md [hunk 7]
```diff
@@ -196,29 +197,29 @@ Sometimes depending on your use, you may need to ensure the variations in the SD
...
+- **Timeout** (The callback will return without waiting until the fetching variation process finishes. The default is 30 seconds)
 
 <Tabs>
 <TabItem value="swift" label="Swift">
 
 ```swift showLineNumbers
 // The callback will return without waiting until the fetching variation process finishes
-let timeout = 1000 // Default is 5 seconds
-let client = BKTClient.get()!
+let timeout = 2000 // Default is 30 seconds
```
Comment (s2mr) [inline docs/sdk/client-side/ios/index.md:208]:
```suggestion
let timeout: Int64 = 2000 // Default is 30 seconds
```

**Comment by cre8ivejp** [comment]
@s2mr, thank you for your review!
I have pushed the fixes. Please take a look when you have time.

**Pull request merged**